src/Features/CSharp/Portable/CodeFixes/ConditionalExpressionInStringInterpolation/CSharpAddParenthesesAroundConditionalExpressionInInterpolatedStringCodeFixProvider.cs

```diff
@@ -64,18 +64,52 @@ private static async Task<Document> GetChangedDocumentAsync(Document document, i
             var openParenthesisPosition = conditionalExpressionSyntaxStartPosition;
             var textWithOpenParenthesis = text.Replace(openParenthesisPosition, 0, "(");
             var documentWithOpenParenthesis = document.WithText(textWithOpenParenthesis);
+
             var syntaxRoot = await documentWithOpenParenthesis.GetRequiredSyntaxRootAsync(cancellationToken).ConfigureAwait(false);
             var nodeAtInsertPosition = syntaxRoot.FindNode(new TextSpan(openParenthesisPosition, 0));
-            if (nodeAtInsertPosition is ParenthesizedExpressionSyntax parenthesizedExpression &&
-                parenthesizedExpression.CloseParenToken.IsMissing)
+
+            if (nodeAtInsertPosition is not ParenthesizedExpressionSyntax parenthesizedExpression ||
+                !parenthesizedExpression.CloseParenToken.IsMissing)
+            {
+                return documentWithOpenParenthesis;
+            }
+
+            return await InsertCloseParenthesisAsync(
+                documentWithOpenParenthesis, parenthesizedExpression, cancellationToken).ConfigureAwait(false);
+        }
+
+        private static async Task<Document> InsertCloseParenthesisAsync(
```

Comment:
Do we need to add tests to cover this change?

Reply:
there are many :)

Reply:
I saw tests where formatting was adjusted, but not new tests for situation with newline. That's why I was asking

Reply:
ah gotcha.  we don't have new tests there.  THey're already covered by existing tests.  I'll find the names for you of examples of this.

Reply:
TestAddParenthesesWhileTyping1-5 and TestAddParenthesesWithCS1026PresentBeforeFixIsApplied1-3

Thanks!

Reply:
That's fine. No need, just knowing we have them is enough :-)